openapi_python_client/templates/endpoint_module.py.jinja

```diff
@@ -40,10 +44,12 @@ def _get_kwargs(
     files = {}
     data = {}
     for key, value in multipart_data.to_dict().items():
-         if isinstance(value, File):
+         if is_file(value):
              files[key] = value
-         else:
+         elif isinstance(value, str):
              data[key] = value
+         else:
+             data[key] = json.dumps(value)
```

Comment:
I changed my mind (😅) I think the follwing code will use the wrong content-type:
```
 data[key] = json.dumps(value)
```

It should be :
```
 file[key] = (None, json.dumps(value). 'application/json')
```

Reply:
The httpx docs say:
> If you need to include non-file data fields in the multipart form, use the data=... parameter:

But I think you're right, this way makes more sense and it actually allows us to write cleaner code in the jinja template. Thanks!

Reply:
Also, just thinking about it, we should use 
```
json.dumps(value.to_dict())
```

To avoid to serialize `UNSET` values as for https://github.com/triaxtec/openapi-python-client/pull/430

@csymeonides-mf